bool start
string scene_point_cloud_path
string object_point_cloud_path
---
bool success